api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -1018,6 +1018,16 @@ public void firstMentionOfICD9WithModifiersOrSnomed5DaysAfterICD10WithModifiers(
         controller.countParticipants(cdrVersion.getCdrVersionId(), searchRequest), 1);
   }
 
+  @Test
+  public void countSubjectsForSurveyWithAgeModifiers() {
+    SearchRequest searchRequest =
+        createSearchRequests(
+            DomainType.SURVEY.toString(), ImmutableList.of(survey()), Arrays.asList(ageModifier()));
```

Comment:
nit: I would use ImmutableList instead of Arrays
```suggestion
            DomainType.SURVEY.toString(), ImmutableList.of(survey()),  ImmutableList.of(ageModifier()));
```